XRTK.Lumin/Packages/com.xrtk.lumin/LuminApi.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if PLATFORM_LUMIN
+
+using System;
+using System.Runtime.InteropServices;
+using UnityEngine;
+using UnityEngine.Experimental.XR;
+using UnityEngine.XR.MagicLeap;
+
+namespace XRTK.Lumin
+{
+    /// <summary>
+    /// Shamelessly lifted from the UnityEngine.XR.MagicLeap packages, but Unity had to make their class internal. Boo.
+    /// </summary>
+    internal static class LuminApi
+    {
+        private const string UNITY_MAGIC_LEAP_DLL = "UnityMagicLeap";
```

Comment:
Is there anyway this might have a different name?

Reply:
Nope. It's the dll found in the `com.unity.xr.magicleap` package.